presto-main/src/main/java/com/facebook/presto/execution/resourceGroups/InternalResourceGroup.java

```diff
@@ -110,6 +110,12 @@
     @GuardedBy("root")
     private long hardCpuLimitMillis = Long.MAX_VALUE;
     @GuardedBy("root")
+    private long perQueryMaxExecutionTimeMillis = Long.MAX_VALUE;
+    @GuardedBy("root")
+    private long perQueryTotalMemoryLimitBytes = Long.MAX_VALUE;
```

Comment:
I am guessing that these will go away based on our discussion

Reply:
correct